src/deployments/inputs/HorizonPhaseOneListing.sol

```diff
@@ -0,0 +1,291 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import {ACLManager} from '../../contracts/protocol/configuration/ACLManager.sol';
+import {IPoolConfigurator} from '../../contracts/interfaces/IPoolConfigurator.sol';
+import {MarketReport} from '../interfaces/IMarketReportTypes.sol';
+import {IAaveV3ConfigEngine as IEngine} from '../../contracts/extensions/v3-config-engine/IAaveV3ConfigEngine.sol';
+import {ConfiguratorInputTypes} from '../../contracts/protocol/libraries/types/ConfiguratorInputTypes.sol';
+import {EngineFlags} from '../../contracts/extensions/v3-config-engine/EngineFlags.sol';
+import {AaveV3Payload} from '../../contracts/extensions/v3-config-engine/AaveV3Payload.sol';
+
+contract HorizonPhaseOneListing is AaveV3Payload {
+  address public immutable ATOKEN_IMPLEMENTATION;
+  address public immutable RWA_ATOKEN_IMPLEMENTATION;
+  address public immutable VARIABLE_DEBT_TOKEN_IMPLEMENTATION;
+
+  ACLManager public immutable ACL_MANAGER;
+  IPoolConfigurator public immutable CONFIGURATOR;
+
+  address public immutable GHO_ADDRESS;
+  address public immutable GHO_PRICE_FEED;
+
+  address public immutable USDC_ADDRESS;
+  address public immutable USDC_PRICE_FEED;
+
+  address public immutable RLUSD_ADDRESS;
+  address public immutable RLUSD_PRICE_FEED;
+
+  address public immutable USTB_ADDRESS;
+  address public immutable USTB_PRICE_FEED;
+
+  address public immutable USCC_ADDRESS;
+  address public immutable USCC_PRICE_FEED;
+
+  address public immutable USYC_ADDRESS;
+  address public immutable USYC_PRICE_FEED;
+
+  bytes32 public constant POOL_ADMIN_EMERGENCY_ADMIN_ROLE = keccak256('EMERGENCY_ADMIN');
+
+  constructor(MarketReport memory report) AaveV3Payload(IEngine(report.configEngine)) {
+    ATOKEN_IMPLEMENTATION = report.aToken;
+    RWA_ATOKEN_IMPLEMENTATION = report.rwaAToken;
+    VARIABLE_DEBT_TOKEN_IMPLEMENTATION = report.variableDebtToken;
+
+    ACL_MANAGER = ACLManager(report.aclManager);
+    require(report.poolConfiguratorProxy == address(CONFIG_ENGINE.POOL_CONFIGURATOR()));
+    CONFIGURATOR = IPoolConfigurator(report.poolConfiguratorProxy);
+
+    GHO_ADDRESS = 0x40D16FC0246aD3160Ccc09B8D0D3A2cD28aE6C2f;
+    GHO_PRICE_FEED = 0xD110cac5d8682A3b045D5524a9903E031d70FCCd;
+
+    USDC_ADDRESS = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+    USDC_PRICE_FEED = 0x8fFfFfd4AfB6115b954Bd326cbe7B4BA576818f6;
+
+    RLUSD_ADDRESS = 0x8292Bb45bf1Ee4d140127049757C2E0fF06317eD;
+    RLUSD_PRICE_FEED = 0x26C46B7aD0012cA71F2298ada567dC9Af14E7f2A;
+
+    USTB_ADDRESS = 0x43415eB6ff9DB7E26A15b704e7A3eDCe97d31C4e;
+    USTB_PRICE_FEED = 0x63908721c77a27689fb0f5db78983135ddfBa56f;
+
+    USCC_ADDRESS = 0x14d60E7FDC0D71d8611742720E4C50E7a974020c;
+    USCC_PRICE_FEED = 0x889f5Feb776Ed122e774D8859134b487c7A85B5d;
```

Comment:
i think we can already deploy the scaled price adaptor & use here bc some of these price feeds are incompatible 

also the tests id run against this are also the test_e2e() in v3 proposals repo, not just the internal suite here